### DISCUSSION TITLE
Remove 'Reset' button

### DISCUSSION THREAD

**Discussion by sys13** [epicweb-dev/epic-stack#597] [Ideas]
I was looking through mdn web docs and came across this ([link](https://developer.mozilla.org/en-US/docs/Learn/Forms/Your_first_form)):
> A click on a reset button resets all the form widgets to their default value immediately. From a UX point of view, this is considered bad practice, so you should avoid using this type of button unless you really have a good reason to include one.

And on ([link](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/reset)):
> Note: You should usually avoid including reset buttons in your forms. They're rarely useful, and are instead more likely to frustrate users who click them by mistake (often while trying to click the [submit button](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input/submit)).

So, maybe we should just remove the Reset buttons. Not only is it a UX win, it's also a small simplification of the features/code.

**Comment by kentcdodds** [comment]
Yeah, I think we can remove it. Feel free to open a PR :)

Priority is low because people delete that form when building their own apps anyway.
